src/components/makeResponsive.js

```diff
@@ -89,6 +90,24 @@ function makeResponsive(
             this.setState({ size: this.getSize(width) });
         }, 500);
 
+        /**
+         * Callback function for setting the ref which measureRef is attached to
+         *
+         * @return {void}
+         */
+        innerRef = el => {
```

Comment:
use React.createRef

Reply:
I had to use the callback due to how react-measure is implemented. It has to be a function. Theres an open issue for it https://github.com/souporserious/react-measure/issues/114